lib-es5/utils/index.js

```diff
@@ -1103,7 +1103,7 @@ function verifyNotificationSignature(body, timestamp, signature) {
   var valid_for = arguments.length > 3 && arguments[3] !== undefined ? arguments[3] : 7200;
 
   // verify that signature is valid for the given timestamp
-  if (timestamp < Date.now() - valid_for) {
```

Comment:
Notice that without the fix, this if is always `false` when timestamp is in seconds (which it is)
Also, the jsdoc mention `valid_for` is in seconds and `Date.now()` returns milliseconds

Reply:
Thanks for the contribution:)